codebase-pipelines/buildspec-deploy.yml

```diff
@@ -3,16 +3,67 @@ version: 0.2
 env:
   variables:
     DEPLOY_TIMEOUT: 1800
+  parameter-store:
+    SLACK_TOKEN: /codebuild/slack_oauth_token
 
 phases:
   install:
     commands:
+      - echo "Starting deployment script for ${SERVICE} service"
       - pip install yq --quiet
+      - echo "Installing platform-helper"
+      - pip install dbt-platform-helper --quiet
+      - platform-helper --version
+      - echo "Installing regclient"
+      - curl -s -L https://github.com/regclient/regclient/releases/latest/download/regctl-linux-amd64 > /usr/local/bin/regctl
+      - chmod +x /usr/local/bin/regctl
 
   build:
     commands:
       - set -e
 
+      # Extract timestamp from image config and check if it exists
+      - aws ecr get-login-password --region ${AWS_REGION} | docker login --username AWS --password-stdin ${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_REGION}.amazonaws.com
+
+      # construct Slack message env vars
+      - |
```

Comment:
I think in general, these multiline commands should be reserved for blocks, such as if/then/else clauses, function definitions while loops etc. So I'd put line 30 in it's own single line item.